api.md

```diff
@@ -25,6 +25,24 @@
 > - [show_tablespaces](#show_tablespaces)
 > - [time_bucket](#time_bucket)
 
+## TimescaleDB informational views [](timescaledb_information)
+
+## timescaledb_information.hypertable [](timescaledb_information.hypertable)
+
+Information about hypertables
+
+|Name|Description|
+|---|---|
+| `schema_name` | Schema name of the hypertable. |
+| `table_name` | Table name of the hypertable. |
+| `table_owner` | Owner of the hypertable. |
+| `num_dimensions` | Number of dimensions. |
+| `num_chunks` | Number of chunks. |
+| `table_bytes` |Disk space used by hypertable |
+| `index_bytes` |Disk space used by indexes|
+| `toast_bytes` |Disk space of toast tables|
+| `total_bytes` |Total disk space used by the specified table, including all indexes and TOAST data|
+
```

Comment:
Could we add sample use? 

Get all hypertables. SELECT * ....

Check if a table is a hypertable. SELECT * from ... where table_name = ...

Reply:
Agree with Diana, just to make clear that this is a "view" that one could query through a normal SQL.  Probably obvious to Sven, might not be to everybody.

Reply:
Yes, there should be a "Sample Usage" section.